### DISCUSSION TITLE
Movie doesn't start, published via Wistia in a Vulcan V2 player

### DISCUSSION THREAD

**Discussion by ralf-deltion** [SafeExamBrowser/seb-win-refactoring#792] [General]
Hello,

Do you know if Vulcan V2 Player of Wistia is supported in Safe Exam Browser?

Thank yoy.

Kind regards,
Ralf

**Comment by dbuechel** [comment]
If it is an HTML5 player, it should theoretically work. But it's best if you quickly try it out yourself using SEB...

**Reply by dbuechel** [reply]
Please do note that proprietary audio- and video-codecs do not work with SEB 3.x for Windows, this due to licensing issues in the browser engine we're using (CEF resp. Chromium).

**Comment by ralf-deltion** [comment]
Thank you for your response. I already tested it and it doesn't play the video. I don't think it's a HTML5 player.
